src/processors.test.ts

```diff
@@ -19,7 +19,7 @@ const untrackedFilename = "an-untracked-file.js";
 
 const gitMocked: jest.MockedObjectDeep<typeof git> = jest.mocked(git);
 gitMocked.getDiffFileList.mockReturnValue([filename]);
-gitMocked.getUntrackedFileList.mockReturnValue([untrackedFilename]);
+gitMocked.getTrackedFileList.mockReturnValue([filename, "file-with-dirty-index.js"]);
```

Comment:
One of the tests also works with a file called `file-with-dirty-index.js` so we have to add it to the "tracked" list too.